docs/getting-started.md

```diff
@@ -224,42 +270,55 @@ One can not fix the sources of already released package artifacts so to resolve
       path: "acorn"
 ```
 
-*Important*: Any time you add a [VCS curation][ort-curations] to [ort-config] you will need do a re-scan of your project. Once the re-scan has completed you should see a reduced number of detected licenses.
+*Important*: Any time you add a [VCS curation][ort-curations] to [ort-config] you need to re-scan your
+project. Once the re-scan has completed you should see a reduced number of detected licenses. 
 
-### Tips for Working with curations
+### Tips for Working with Curations
 
-- Fix incorrect package metadata upstream if the issue exists for latest version. Contributing back helps the community and you; once package maintainers are aware of the issue they might fix the same issue for all the packages they maintain.
-- Contribute when possible curations you make to the [ort-config] repository so any [ORT][ort] user can benefit
+- Fix incorrect package metadata upstream if the issue exists in the latest version. Contributing back helps both you and the
+  community; once package maintainers are aware of the issue, they can fix it for all the packages they maintain. 
+- When possible, contribute the curations you make to the [ort-config] repository so that any [ORT][ort] user can benefit.
 
-## Marking Files or Directories in the Sources of a Dependency as not Included in Released Artifacts <a name="orthw-pc-excludes"></a>
+## Marking Files or Directories in Dependency Sources as not Included in Released Artifacts <a name="orthw-pc-excludes"></a>
 
-Using [package configuration][ort-package-configurations] you can mark files and directories as not included in released artifacts - use it to make clear that license findings in documentation or tests in a package sources do not apply to the release (binary) artifact which is a dependency in your project.
+A [package configuration][ort-package-configurations] allows you to mark files and directories as not included in
+released artifacts - this makes it clear that license findings in documentation or tests in the package sources do not
+apply to the release (binary) artifact that is a dependency in your project.
 
-[Generate the Web App report](#orthw-report-webapp) and look up `NPM::acorn-jsx:5.2.0` by filtering the _Package_ column under _Table_. You will find the declared license is MIT but the detected are BSD-2-Clause and MIT so which licenses apply?
+[Generate the Web App report](#orthw-report-webapp) and look up `NPM::acorn-jsx:5.2.0` by filtering the _Package_ column
+under _Table_. You will find that the declared license is MIT, but the detected licenses are BSD-2-Clause and
+MIT. Therefore, it is necessary to determine which licenses apply.
```

Comment:
When reading "Therefore, it is necessary to determine which licenses apply." I was wondering "why" it is "necessary".

e.g. determining that might not be necessary depending on the users way of working.

Reply:
Changed